src/main/scala/nl/knaw/dans/api/sword2/DepositHandler.scala

```diff
@@ -160,6 +166,38 @@ object DepositHandler {
     }
   }
 
+  private def resolveFetchItems(bagitDir: File)(implicit id: String): Try[Unit] = Try {
+    log.debug(s"[$id] Checking fetch.txt")
```

Comment:
When I encountered this line in the logs, I wasn't really sure if that meant a `fetch.txt` was found. Apparently, it does not mean that, necessarily. I think the message should be clear about that.